src/scss/typography.scss

```diff
@@ -0,0 +1,47 @@
+// Typography
+// to use: @include typo('type of typography', 'font size rem, em, px, %');
+
+$h1-color: #ffffff;
```

Comment:
Is this naming a little too specific? Like, what if I want to align the colour of my `p` and my `h1`, do I use `$h1-color` for both? Is there a more generic naming convention you can use? If you message me, I can suggest what I do.